tests/test_analytical_solution.py

```diff
@@ -0,0 +1,75 @@
+"""Testing against the analytical solution by Iskhakov et al (2017)."""
+import pickle
+from pathlib import Path
+
+import numpy as np
+import pytest
+from lcm.entry_point import get_lcm_function
+from lcm.example_models import PHELPS_DEATON_NO_BORROWING
+from numpy.testing import assert_array_almost_equal as aaae
+
+# Path to analytical solution
+DATA = Path(__file__).parent.resolve() / "analytical_solution"
+
+
+def numerical_solution(params):
+    """Numerical solution."""
+    model = PHELPS_DEATON_NO_BORROWING
+    model["n_periods"] = params["num_periods"]
+    model["choices"]["consumption"]["start"] = 1
+    model["choices"]["consumption"]["stop"] = 100
+    model["choices"]["consumption"]["n_points"] = 10_000
+    model["states"]["wealth"]["start"] = 1
+    model["states"]["wealth"]["stop"] = 100
+    model["states"]["wealth"]["n_points"] = 10_000
```

Comment:
Can you write these parameters in a dict and then update the model like:

```python
config_update = {...}
model = {**PHELPS_DEATON_NO_BORROWING, **config_update}
```

Can you use `n_periods` instead of `num_periods`?

Reply:
Done. I haven't found a way to really only update the values that I'm trying to change. The `retirement` dict is, for instance, part of the `choices` dict and I have to explicitly add it in the `config_update` dict, event though I leave it unchanged compared to the version in `PHELPS_DEATON_NO_BORROWING`.